### PR TITLE
Feat: move kube-trigger addon to official addon

### PR DESCRIPTION
**Pull request by FogDong**
Signed-off-by: FogDong <dongtianxin.tx@alibaba-inc.com>

### Checklist

I have:

- [ ] Title of the PR starts with type (e.g. `[Addon]` , `[example]` or `[Doc]`).
- [ ] Updated/Added any relevant [documentation](https://kubevela.io/docs/reference/addons/overview) and [examples](https://github.com/kubevela/catalog/tree/master/examples).
- [ ] New addon should be put in [experimental](https://github.com/kubevela/catalog/tree/master/experimental/addons).
- [ ] Update addon should modify the `version` in `metadata.yaml` to generate a new version.

####  Verified Addon promotion rules

If this pr wants to promote an experimental addon to verified, you must check whether meet these conditions too:
  - [ ] This addon must be tested by addon's [e2e-test](./test/e2e-test/addon-test) to guarantee this addon can be enabled successfully.
  - This addon must have some basic but necessary information.
    - [ ] An accessible icon url and source url defined in addon's `metadata.yaml`.
    - [ ] A detail introduction include a basic example about how to use and what's the benefit of this addon in `README.md`.
    - [ ] Also provide an introduction in KubeVela [documentation](https://kubevela.net/docs/reference/addons/overview).
    - [ ] It's more likely to be accepted if useful examples provided in example [dir](examples/).

### CODE REVIEW

File: addons/fluxcd/definitions/helm-release-def.cue [hunk 1]
```diff
@@ -11,43 +11,42 @@ helm: {
...
-													releaseMessage: releaseBasicMessage
-												}
-												if len(releaseStatus.conditions) > 1 {
-													if releaseStatus.conditions[1]["message"] != _|_ {
-														releaseMessage: releaseBasicMessage + ", " + releaseStatus.conditions[1]["message"]
-													}
-												}
-										}
-									}
+					if context.output.status != _|_ if context.output.status.conditions == _|_ {
```
Comment (wonderflow) [inline addons/fluxcd/definitions/helm-release-def.cue:34]:
```suggestion
					if context.output.status != _|_ if context.output.status.conditions != _|_ {
```

File: addons/kube-trigger/resources/crd.cue [hunk 2]
```diff
@@ -0,0 +1,228 @@
+package main
+
+crd: {
+	name: "workflowrun-crd"
```
Comment (charlie0129) [inline addons/kube-trigger/resources/crd.cue:4]:
```suggestion
	name: "kubetrigger-crd"
```

**Review by wangyikewxgm — DISMISSED** [review]

File: addons/kube-trigger/metadata.yaml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 name: kube-trigger
-version: 0.0.1-alpha.2
+version: 1.0.0
 description: kube-trigger watches events and triggers actions.
 url: https://github.com/kubevela/kube-trigger
```
Comment (wonderflow) [inline addons/kube-trigger/metadata.yaml:5]: please add a kubevela logo here

File: addons/kube-trigger/parameter.cue [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+	imagePullPolicy: *"IfNotPresent" | "Never" | "Always"
+	version:         *"v0.1.0" | string
+	resources: {
+		requests: {
+			cpu:    *"10m" | string
+			memory: *"64Mi" | string
+		}
+		limits: {
+			cpu:    *"500m" | string
+			memory: *"128Mi" | string
```
Comment (wonderflow) [inline addons/kube-trigger/parameter.cue:21]: make it larger

1000m
1G

**Review by wonderflow — APPROVED** [review]

**Pull request merged**